corehq/apps/hqcase/models.py

```diff
@@ -1,6 +1,5 @@
-# This file is only here so that django will recognize that 
+# This file is only here so that django will recognize that
 # this is a valid app and run the associated unit tests.
```

Comment:
If this comment is still true then I think this file can be removed since we now use nose and nose will find tests anywhere, not just in django apps.